sources/packages/web/src/services/http/Institution.ts

```diff
@@ -195,4 +197,42 @@ export class InstitutionApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getPaginatedAESTInstitutionProgramsSummary(
+    institutionId: number,
+    pageSize: number,
+    page: number,
+    sortColumn: string,
+    sortOrder: SortDBOrder,
+    searchName: string,
+  ): Promise<AESTInstitutionProgramsSummaryPaginatedDto> {
+    const sortByOrder = sortOrder === SortDBOrder.ASC ? "ASC" : "DESC"; //Default sort order
+    try {
+      let queryString = "";
+      if (searchName) {
```

Comment:
By adding & in front we can avoid string.slice(0,-1)

![image](https://user-images.githubusercontent.com/54600590/145456114-7e20ceec-5926-4166-b1d8-27165e9bc5b0.png)